server/models/Tier.js

```diff
@@ -88,13 +106,13 @@ export default function(Sequelize, DataTypes) {
             } else if (this.maxQuantity) {
               return this.maxQuantity;
             } else {
-              return -1; // means there was no maxQuantity set, so infinite availability
+              return Infinity;
```

Comment:
Infinity is a supported JS concept. Only place it's used is in Graphql types right now. If needed elsewhere, we can make it a constant.